words_dir/tweet_labels.py

```diff
@@ -36,7 +36,7 @@
     'straight': 'Heterosexual, sexually and/or emotionally attracted to members of the opposite sex.',
     'aegosexual': 'lacking any desire to be a participant in the sexual activities therein',
     'abrosexual': 'Abrosexual refers to an individual whose sexuality is changing or fluid. For example, someone could be gay one day, then be asexual the next, then polysexual the next.',
-
+    'aromatic': 'An adjective used to describe the people who do not experience romantic attraction.',
```

Comment:
Thanks for adding this. Would you mind fixing up the spelling of `aromantic`? 

That is, unless you meant 
`'aromatic': 'One who has a pleasant or distinct smell.',` lol

Reply:
Yes, i will fix up the spelling